projects/noloan/app/src/main/java/hasadna/noloan/SpamRecyclerAdapter.java

```diff
@@ -29,26 +31,80 @@
     extends RecyclerView.Adapter<SpamRecyclerAdapter.RecyclerViewHolder> {
   private static final String TAG = "SpamRecyclerAdapter";
 
+  ArrayList<SmsMessage> inboxSpam;
+
   public SpamRecyclerAdapter() {
     Handler handler = new Handler(Looper.myLooper());
+    inboxSpam = new ArrayList<>();
+
+    // Fetch spams from DB that are in user's inbox
+    for (int i = 0; i < SmsMessages.get().getInboxMessages().size(); i++) {
+      for (SmsMessage sms : SmsMessages.get().getDbMessages()) {
+        if (SmsMessages.get().getInboxMessages().get(i).getBody().contentEquals(sms.getBody())
```

Comment:
Why didn't you use 'searchMessage'?

Reply:
Good point, forgot about it. Thanks
Fixed :+1: